lib/services/flowdock.ts

```diff
@@ -245,22 +299,34 @@ export class FlowdockService extends MessageService implements ServiceEmitter, S
 }
 
 /**
- * Return this class activated and typed as a listener
+ * Build this class, typed and activated as a listener
+ * @returns {ServiceListener}
  */
 export function createServiceListener(): ServiceListener {
     return new FlowdockService(true);
 }
 
 /**
- * Return this class typed as an emitter
+ * Build this class, typed as an emitter
+ * @returns {ServiceEmitter}
  */
 export function createServiceEmitter(): ServiceEmitter {
     return new FlowdockService(false);
 }
 
 /**
- * Return this class typed as a messenger
+ * Build this class, typed as a message service
+ * @returns {MessageService}
  */
 export function createMessageService(): MessageService {
     return new FlowdockService(false);
 }
+
+//noinspection JSUnusedGlobalSymbols
```

Comment:
My IDE (I've gone back to IntelliJ) is "clever" enough to think that this isn't used, because it can't track through
```
// Create and store the hub service
const hub = require(`../services/${process.env.SYNCBOT_HUB_SERVICE}`);
this.hub = hub.createDataHub();
```
This is me saying, I know, don't tell me.